README.md

```diff
@@ -159,9 +159,7 @@ __NOTES:__
 
 - `ansible_delegate_hostname` must match `inventory_hostname` in ansible inventory list.
 Some tasks will be delegated from backup server to this client
-- `enable_backup_job` - Will create backup job `DefaultJobLinux`
```

Comment:
What is the purpose of this parameter removal? What issue does it solve?

Reply:
The corresponding task was only used to create the job config file (and forcing its name), while we're already handling it somewhere else. It's way better imo to enable or disable a job in it's config while (that's why I added the possibility to the template), but always copy the job config file if the client's registered.

Reply:
I would say `enable_backup_job: true` allows to create a simple backup jobs for the clients out of the box. It's a nice feature for people with basic needs. However, if you feel this might be misleading for the users, then go for removal.

Reply:
This is a breaking change you will have to document though.